journalpump/rsyslog.py

```diff
@@ -96,7 +96,8 @@ def __init__(
         if protocol is None:
             protocol = "PLAINTEXT"
         if cacerts is not None or protocol == "SSL":
-            self.ssl_context = ssl.SSLContext(ssl.PROTOCOL_TLS)
+            self.ssl_context = ssl.create_default_context()
```

Comment:
This calls `load_default_certs()`. I'm guessing that when `certfile` is present we should only accept what's in there but will that be merged with the default certs so that we accept certs that should not be accepted?